test/over_react/component/ref_util_test.dart

```diff
@@ -60,7 +60,7 @@ main() {
           final Ref refObject = createRef();
           final vDomElement = (BasicForwarded()..ref = refObject)();
 
-          expect(getProperty(getProperty(vDomElement.type, 'render'), 'displayName'), 'Anonymous');
+          expect(getProperty(getProperty(vDomElement.type, 'render'), 'name'), '');
```

Comment:
1. forwardRef2 uses `name`, not `displayName`
2. An empty string is used instead of `Anonymous` in `forwardRef2` (the empty string shows up as "Anonymous" in the dev tools, so functionality is equivalent), so these needed to be updated